README.md

```diff
@@ -67,6 +68,17 @@ helm install ratify \
     --set-file notaryCert=./notary.crt
 ```
 
+Option 2: Install ratify with charts from your local branch.  
+Note: Latest chart in main may not be compatible with the last released version of ratify image. Weekly dev build coming soon. Please check back on issue [643](https://github.com/deislabs/ratify/issues/643)
+```bash
+helm install ratify \
```

Comment:
probably we should copy
```
helm repo add ratify https://deislabs.github.io/ratify
# download the notary verification certificate
curl -sSLO https://raw.githubusercontent.com/deislabs/ratify/main/test/testdata/notary.crt
```
 here as well since users may just skip option 1 steps.

Reply:
+1, we should add those two steps in option 2.

Reply:
thanks, added clone steps.

Reply:
> probably we should copy
> 
> ```
> helm repo add ratify https://deislabs.github.io/ratify
> # download the notary verification certificate
> curl -sSLO https://raw.githubusercontent.com/deislabs/ratify/main/test/testdata/notary.crt
> ```
> 
> here as well since users may just skip option 1 steps.

@binbin-li , since they have cloned the repository, they shouldn't need to get the helm repo or download the notary.crt. Correct?

Reply:
Do you think this should point to the new dev builds we will have as the repository and tag fields of the helm chart? Right now it will use the latest release

Reply:
added a link to the dev build in the release section. thanks!